Assets/MixedRealityToolkit/README.md

```diff
@@ -0,0 +1,28 @@
+# Mixed Reality Toolkit
+
+This folder contains all the individual features of the MRTK, providing systems to assist in VR/XR/AR solutions
+The Core folder is required and should not be removed, whereas the other feature folders are completely optional. Wither use the versions provided by the toolkit, or roll your own.
+
+# In this Volder
```

Comment:
Volder -> Folder

Reply:
Rofl, my faux German filter must have kicked in :D